wd_capabilities.au3

```diff
@@ -284,58 +285,67 @@ EndFunc   ;==>_WD_CapabilitiesAdd
 ; ===============================================================================================================================
 Func _WD_CapabilitiesGet()
 	Local $Data1 = Json_Encode($_WD_CAPS__OBJECT)
-	
+
 	Local $Data2 = Json_Decode($Data1)
 	Local $Json2 = Json_Encode($Data2, $Json_UNQUOTED_STRING)
-	
+
 	Local $Data3 = Json_Decode($Json2)
 	Local $Json3 = Json_Encode($Data3, $Json_PRETTY_PRINT, "    ", ",\n", ",\n", ":")
-	
+
 	Return $Json3
 EndFunc   ;==>_WD_CapabilitiesGet
 
 ; #FUNCTION# ====================================================================================================================
-; Name ..........: _WD_CapabilitiesNewType
+; Name ..........: _WD_CapabilitiesNew
```

Comment:
I'm not sure if it should be named:
`_WD_CapabilitiesNew` or `_WD_CapabilitiesNewType`

Reply:
What is your opinion on the name of this Function

Reply:
I think you should pick whichever one best describes the action being performed. A few others to consider are `_WD_CapabilitiesCustom` or `_WD_CapabilitiesDefine`

Reply:
`_WD_CapabilitiesDefine` is the best